panacea/grpc_client.go

```diff
@@ -81,6 +83,23 @@ func (c *GrpcClient) GetAccount(panaceaAddr string) (authtypes.AccountI, error)
 	return acc, nil
 }
 
+func (c *GrpcClient) GetOracleRegistration(oracleAddr, uniqueID string) (*oracletypes.OracleRegistration, error) {
+	client := oracletypes.NewQueryClient(c.conn)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	reqMsg := &oracletypes.QueryOracleRegistrationRequest{
+		UniqueId: uniqueID,
+		Address:  oracleAddr,
+	}
+	response, err := client.OracleRegistration(ctx, reqMsg)
+	if err != nil {
+		return &oracletypes.OracleRegistration{}, err
```

Comment:
I think it would be good for OracleRegistration to return nil when an error occurs.
In general, when an error occurs, it is thought that only the error is checked and not the object that responded. :)

```suggestion
		return nil, err
```

Reply:
ty. I changed it in [c9e91fc](https://github.com/medibloc/panacea-doracle/pull/13/commits/c9e91fc4af0152cff0421497127f61ff92c3ddf0)